network-api/networkapi/campaign/views.py

```diff
@@ -31,7 +31,7 @@ def send_message(self, QueueUrl, MessageBody):
         }
 
 
-# Basket/Salesforce SQS client
+# Basket/Salesforce SQS client  ##### UPDATE?
```

Comment:
Does this need to be updated at all? I can't tell 😬

Reply:
no, we still need SQS for petitions.